dlls/ff/ff_lualib_team.cpp

```diff
@@ -62,8 +86,11 @@ void CFFLuaLib::InitTeam(lua_State* L)
 		class_<CFFTeam, CTeam>("Team")
 			.def("SetAllies",			&CFFTeam::SetEasyAllies)
 			.def("ClearAllies",			&CFFTeam::ClearAllies)
+			.def("GetAllies",			&FFLib::GetAllies)

```

Comment:
I notice that the other functions here are tagged with CTeam or CFFTeam, but this is FFLib, whilst being under `class_<CFFTeam, CTeam>("Team")`, is this OK?